docs/manual/manual_safety_systems.rst

```diff
@@ -0,0 +1,174 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_safety:
+
+*****************
+Safety Systems
+*****************
+The following manual is meant to outline the safety systems present on Vertiq modules. Safety systems are built into your module's firmware, 
+and seek to protect your module from things like velocity runaway, power issues, and overheating.
+
+Protecting Against Voltage Regeneration Spikes
+====================================================
+Voltage regeneration is one of the most often cited reasons for module failure, and as such we have advanced systems to protect against extreme voltage spikes. 
+You can read more about voltage regeneration protection in :ref:`protect_against_regen`.
+
+Protection with Derates
+===========================
+Basically, a derate is a scaling factor that adjusts a control parameter into a safer area of operation. All derate values on Vertiq's modules are represented as float 
+values from [0,1].
+
+Suppose that your :ref:`commanded drive voltage <manual_velocity_control_mechanisms>` is 2V, and the current derate is 0.4. 
+The actual drive voltage applied will be (2V * 0.4) = 0.8V. A basic illustration of derate application is shown below
+
+.. image:: ../_static/manual_images/safety/basic_derate_example.png
+    :scale: 50%
+
+There are three separate derate values that are internally updated, each linked with a different module function. There is a speed derate used to avoid runaway, 
+a microcontroller temperature based derate to avoid microcontroller overheating, and a coil temperature derate to avoid coil overheating. 
+These three derate values are combined and applied to the target command voltage in order to protect your module. Details of each of these derate values as well as 
+how they are combined are discussed below.
+
+Speed Based Derate
+---------------------
+As mentioned above, your module's speed derate is meant to protect your module against speed runaway. Speed derate is calculated using the module's current speed, 
+brushless drive's ``motor_redline_start`` parameter, and brushless drive's ``motor_redline_end`` parameter. You can find the redline parameters listed in the 
+:ref:`Brushless Drive Message Table <brushless_drive>`.
+
+The total speed derate is calculated by :math:`\text{Speed Derate} = \frac{\text{motor_redline_end - speed}}{\text{motor_redline_end - motor_redline_start}}` where the solution is saturated to [0,1].
+
+This means that for any speed below the range [redline_start, redline_end], there is no derate applied. For any speed above the range [redline_start, redline_end], 
+the speed derate is 0. Any speed in that range is subject to a variable derate [0, 1].
+
+Suppose you set your redline start to 1750 rad/s and your redline end to 2000 rad/s. This means that there will be no derate limiting applied under 1750 rad/s, 
+your module will stop spinning at and above 2000 rad/s, and will be derate limited in between.
```

Comment:
We really ought to go through the control center and pull out things that "customer's shouldn't generally be using." If we really really don't want them touching things, we shouldn't give a big button to change it (even if it's behind the very scary advanced tab). If someone comes to us with an issue, and we determine it can be fixed by changing the redlines, then we can just tell them how to do it with the API.